docs/pages/desktop-access/getting-started.mdx

```diff
@@ -18,6 +18,17 @@ The features documented on this page are available in Teleport `10.2.6` and high
 This guide will help you configure Teleport and Active Directory in order to
 provide secure, passwordless access to Windows desktops.
 
+<Details title="Compare Desktop Access to traditional RDP">
+
+Teleport Desktop Access is desiged to be a secure access protocol for Windows
```

Comment:
```suggestion
Teleport Desktop Access is desiged to be a secure access solution for Windows
```

Reply:
Incorporated in https://github.com/gravitational/teleport/pull/18655/commits/7152a21cee510e763e56d80b8f8f7cf39247a1ec